r/R/filesystem.R

```diff
@@ -476,15 +476,13 @@ SubTreeFileSystem$create <- function(base_path, base_fs = NULL) {
 #' copying but may help accommodate high latency FileSystems.
 #' @return Nothing: called for side effects in the file system
 #' @export
-#' @examples
-#' \dontrun{
+#' @examplesIf FALSE
```

Comment:
I think it's best to keep `\dontrun{}` in cases where the conditional is a literal `FALSE`

Reply:
On second thought, disregard this—I hadn't realized that you eliminated _all_ the other instances of `\dontrun{}` so for consistency it makes sense to do this one too.